### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by averingood**

### CODE REVIEW

File: src/components/particles/particles.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+import Particles from "react-tsparticles";
+
+const ParticlesCustom = () => {
+    return (
+        <>
+            <Particles
+                id="tsparticles"
+                options={{
+                    "particles": {
```
Comment (vectorincorporated) [inline src/components/particles/particles.js:11]: Separate component view and "particles" logic in different files. Create some folder with animation config and import object from this file.

File: src/pages/index.js [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
-    <p>Now go build something great.</p>
-    <div style={{ maxWidth: `300px`, marginBottom: `1.45rem` }}>
-      <Image />
-    </div>
-    <Link to="/page-2/">Go to page 2</Link> <br />
-    <Link to="/using-typescript/">Go to "Using TypeScript"</Link>
-  </Layout>
-)
+    <Layout>
+        <div className="particles-container">
```
Comment (vectorincorporated) [inline src/pages/index.js:9]: Try to use SEO in pages if you can. In this case move SEO from home block to pages/index.js

**Review by vectorincorporated — COMMENTED** [review]
Comments added.

File: package.json [hunk 3]
```diff
@@ -17,7 +17,8 @@
     "prop-types": "^15.7.2",
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
-    "react-helmet": "^6.0.0"
+    "react-helmet": "^6.0.0",
+    "react-tsparticles": "^1.18.6"
```
[the remark this answers is not in the document]
Comment (averingood) [reply package.json:21]: > Why do we need this library?

this is a library with animation for main block

File: src/components/layout.css [hunk 4]
```diff
@@ -31,6 +31,11 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+main {
+  width: 800px;
```
Comment (vectorincorporated) [inline src/components/layout.css:35]: Remove width

File: src/components/particles/particles-options.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
+export const options = {
```
Comment (vectorincorporated) [inline src/components/particles/particles-options.js:1]: Move it to folder like utils or constants

**Review by vectorincorporated — APPROVED** [review]

**Pull request merged**